client/src/dbus/mod.rs

```diff
@@ -55,7 +55,7 @@ pub mod api;
 mod api;
 
 mod algorithm;
-pub(crate) use algorithm::Algorithm;
+pub use algorithm::Algorithm;
```

Comment:
It is not used as public API, make it behind the unstable flag similar to `mod api;` / `pub mod api;`

Reply:
done, thanks!